tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -440,11 +440,13 @@ public function testBug4715(): void
 	public function testBug4734(): void
 	{
 		$errors = $this->runAnalyse(__DIR__ . '/data/bug-4734.php');
-		$this->assertCount(3, $errors);
+		$this->assertCount(5, $errors); // could be 3
```

Comment:
this is a topic for a followup: it seems we miss to emit `PropertyAssignNode` for property assignments in closure-bind scope

Reply:
Yeah of course, static analysis is not fit for these shenanigans 😊 Especially for private properties.